m4_sessions/1811_inning/belong_in_tech.md

```diff
@@ -65,8 +74,13 @@ fill time for character description
 
 
 closing - 
-5 minutes on reflection
-5 minutes on survey
+ reflection questions
```

Comment:
These are great!  Be sure to have these on a slide or on the board so it isn't just people listening and then forgetting.